meta-lmp-bsp/recipes-support/mfgtool-files/mfgtool-files_%.bbappend

```diff
@@ -57,3 +63,25 @@ do_deploy_prepend_apalis-imx6() {
     install -m 0644 ${DEPLOY_DIR_IMAGE}/SPL ${DEPLOYDIR}/${PN}/SPL-mfgtool
     install -m 0644 ${DEPLOY_DIR_IMAGE}/u-boot.itb ${DEPLOYDIR}/${PN}/u-boot-mfgtool.itb
 }
+
+do_compile_append_apalis-imx6-sec() {
+    sed -i '/^FB: ucmd setenv.*/i\
+	\
+	# Clear fiovb vars \
+	FB: ucmd fiovb init 2 \
+	FB[-t 50000]: ucmd fiovb write_pvalue bootcount 0 \
+	FB[-t 50000]: ucmd fiovb write_pvalue rollback 0\n' full_image.uuu
+
+    sed -i 's/SPL.*/&.signed/g' bootloader.uuu
+    sed -i 's/SPL.*/&.signed/g;s/^[ \t]*//g' full_image.uuu
```

Comment:
Just FYI: `sed 's/^[ \t]*//g'` is needed to remove the tabs included by the first sed command, any suggestions to simplify this?

Reply:
you should be able to remove the indent on the initial sed  and the trailing space and then the last filter is not needed.

Reply:
do you mean:
`do_compile_append_apalis-imx6-sec() {
    sed -i -e '/^FB: ucmd setenv.*/i\
\
# Clear fiovb vars \
FB: ucmd fiovb init 2 \
FB[-t 50000]: ucmd fiovb write_pvalue bootcount 0 \
FB[-t 50000]: ucmd fiovb write_pvalue rollback 0\n' full_image.uuu`